Resources/Textures/Mobs/Pets/cakecat.rsi/meta.json

```diff
@@ -0,0 +1,36 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Update by Nyranu. The original PR from TheShuEd",
```

Comment:
you should put here copyright from `Resources/Textures/Mobs/Pets/cat.rsi/meta.json` and new copyright for inhand and equipped states